commercial/app/views/books/booksProminent.scala.html

```diff
@@ -9,7 +9,9 @@
                 <div class="commercial__header__inner">
                     <h3 class="commercial__title">
                         <a href="@{clickMacro}http://www.guardianbookshop.co.uk/" data-link-name="merchandising-bookshop-v@{version}_@{date}-high-visit-shop">
-                            <i class="i i-marque-54"></i><i class="i i-guardian-logo-commercial"></i><i class="i i-bookshop-logo i-commercial-brand"></i>
+                            @fragments.inlineSvg("marque-54", "commercial")
```

Comment:
The line-breaks are fine, code becomes really unwieldy if they are all on one line.

I'll move the marque to icons.